decidim-meetings/app/views/decidim/meetings/layouts/redesigned_live_event.html.erb

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
```

Comment:
Is the live event page working?

It looks broken when I enter it (a white page without any styles). Tried a YouTube embed.

Reply:
The problem here is that the live_event page contains so many legacy warnings, they block most of the visible content. Take a look at a "clear" output for an admin user

![imagen](https://user-images.githubusercontent.com/817526/209320118-036e087c-b802-4844-8abd-bc28c9333555.png)